src/sentry/static/sentry/app/views/projectKeyDetails.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import DocumentTitle from 'react-document-title';
-import underscore from 'underscore';
+import underscore from 'lodash';
```

Comment:
I would also say it makes it less confusing for new people to know where to google documentation